packages/react-native/src/lib/utils.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Filters surveys based on the displayOption, recontactDays, and segments
+ * @param environmentSate - The environment state
+ * @param personState - The person state
+ * @returns The filtered surveys
+ */
+import { diffInDays } from "@formbricks/lib/utils/datetime";
+import {
+  type TJsEnvironmentState,
+  type TJsEnvironmentStateSurvey,
+  type TJsPersonState,
+} from "@formbricks/types/js";
+
+// takes the environment and person state and returns the filtered surveys
+export const filterSurveys = (
```

Comment:
As this is a utils functions can we import it from js-core, so that we wont have to make changes at both the places ?